tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetMvc5.cs

```diff
@@ -72,17 +72,20 @@ public AspNetMvc5(IisFixture iisFixture, ITestOutputHelper output, bool classicM
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
         [Trait("LoadFromGAC", "True")]
-        [SkippableTheory]
-        [InlineData("/Health/?test&[$slice]")]
-        [InlineData("/Health/wp-config")]
-        [InlineData]
-        public Task TestSecurity(string url = DefaultAttackUrl)
+        [Theory]
```

Comment:
We should use `[SkippableTheory]` in all integration tests, to cope with known flaky behaviour

Reply:
I then Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2495/commits/6785f56e8af6d600e01818116fa36b76c7e0cdb6